packages/@vue/cli-plugin-typescript/package.json

```diff
@@ -26,7 +26,7 @@
     "@types/webpack-env": "^1.15.1",
     "@vue/cli-shared-utils": "^4.2.2",
     "cache-loader": "^4.1.0",
-    "fork-ts-checker-webpack-plugin": "^1.5.1",
+    "fork-ts-checker-webpack-plugin": "^4.0.3",
```

Comment:
Could you please revert this to v3.x?
I just checked the changelog and the `compiler` option was supported since v3.1.
Meanwhile, the v4 release came with too many breaking changes, such as removing tslint support, we can't ship it in a minor version.